src/sdformat_urdf_plugin.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include <rcutils/logging_macros.h>
+#include "sdformat_urdf/sdformat_urdf.hpp"
+#include <urdf_parser_plugin/parser.h>
+
+namespace sdformat_urdf {
+
+class SDFormatURDFParser : public urdf::URDFParser
+{
+public:
+
+  SDFormatURDFParser() = default;
+  virtual ~SDFormatURDFParser() = default;
```

Comment:
This doesn't seem like a class that would be subclassed, so I'll suggest marking the class `final` and removing the virtual destructor.

Reply:
Use `final` and remove `virtual` from destructor in 21adf189d3d02e72014e5a642a15123c7ca5cf32